babel.config.js

```diff
@@ -20,6 +20,8 @@ const babelTransforms = new Map([
   ['babel-jest', 'getEmptyConfig'],
   ['post-closure', 'getPostClosureConfig'],
   ['pre-closure', 'getPreClosureConfig'],
+  ['bento-element-minified', 'getBentoElementMinifiedConfig'],
+  ['bento-element-unminified', 'getBentoElementUnminifiedConfig'],
```

Comment:
The minified and unminified configurations already have a bunch of conditional logic within their `plugins`.
If there is a way to gate on Bento, you should be able to just add 1-line to those files instead of the ceremony here and in bento-element-config.js

Something like:
```js
// minified-config.js
plugins: [
  isBento && './build-system/babel-plugins/babel-plugin-bento-imports'
  ...
]
```

Reply:
We usually differentiate by flag at that level, since there is no way to pass options to the babel caller.

Unfortunately, we don't have a separate flag for `--bento`, so we'd need to refactor some parts of CI to run that separately.

I might be wrong about being able to pass `isBento`, would love to hear your thoughts.

Reply:
Good point, will follow up with Justin here later to see if theres any way for us to flexibly pass options into the config-getters.

Reply:
Good point raised by @samouri, agree that it can be solved separately.